### PR TITLE
Moved send_mention_notification to ActiveJob

### PR DESCRIPTION
**Pull request by lightalloy**
## What type of PR is this? (check all applicable)
- [x] Refactor

## Description
- moved send_mention_notification to `ActiveJob`
- used @copasetickid's pr and refined it 

## Related Tickets & Documents
#1996

### CODE REVIEW

File: app/services/notifications/new_mention/send.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      def initialize(mention)
+        @mention = mention
+      end
+
+      def self.call(*args)
+        new(*args).call
+      end
+
+      def call
+        Notification.create(
```
Comment (rhymes) [inline app/services/notifications/new_mention/send.rb]: I don't recall but, can this ever fail?

Comment (lightalloy) [reply app/services/notifications/new_mention/send.rb]: Like everything else :thinking: Probably, `create!` would be better?

Comment (rhymes) [reply app/services/notifications/new_mention/send.rb]: I guess it depends if you want to be notified of the error or not :D

Comment (lightalloy) [reply app/services/notifications/new_mention/send.rb]: Updated that. Noticed several other similar places where I should do this.

**Pull request merged**